teammapper-frontend/mmp/src/map/handlers/draw.ts

```diff
@@ -55,16 +55,26 @@ export default class Draw {
      * Update the dom of the map with the (new) nodes.
      */
     public update() {
-        const nodes = this.map.nodes.getNodes(),
-            dom = {
-                nodes: this.map.dom.g.selectAll('.' + this.map.id + '_node').data(nodes),
-                branches: this.map.dom.g.selectAll('.' + this.map.id + '_branch').data(nodes.slice(1))
-            }
+        let nodes = this.map.nodes.getNodes()
```

Comment:
why let here? do you need to modify this ?

Reply:
Good catch, must've been a mistake from when I separated out nodes and dom variable creation.